README.md

```diff
@@ -192,6 +192,7 @@ $ gcloc languages
   Processing       |  .pde          |  //             |  /* */
   Protocol Buffers |  .proto        |  //             |
   Python           |  .py           |  #              |  """ """
+  PRQL             |  .prql         |  #              |
```

Comment:
Could you add `PRQL` between `Protocol Buffers` and `Python` please ?

Reply:
Done!